api/modules/automations.js

```diff
@@ -33,31 +33,34 @@ const automations = module.exports = (() => {
         const organizations = await getUserOrganizations({
             auth_key: params.auth_key
         })
+
+        let itsOrganization = false
+
+        if (organizations[0].name != params.organizationName) { // Check if its an org or user
+            itsOrganization = true
+        }
+
         const repos = await github.fetchRepos({
-            auth_key: params.auth_key
+            auth_key: params.auth_key,
+            organizationName: params.organizationName,
+            itsOrganization
```

Comment:
I recommend we try to make the language of this logic more clear considering the following..

1. `isOrganization` and not `itsOrganization` because it may be confused with the possessive form "its" which means this organization may belong to some "it"
2. Remember to [minimize require knowledge](https://github.com/setlife-network/trinary/blob/develop/CONTRIBUTING.md#1-minimize-required-knowledge) you want to imagine that you know nothing about this logic before reading it, so as a reader I am asking why you are passing an `organizationName` if it's possible that `isOrganization` will be false? This can be related to @sofiarm21's idea about "Accounts" vs "Organizations" so it is more clear that it is possible for this name to either belong to a GitHub Organization versus a GitHub User